sdks/java/io/pom.xml

```diff
@@ -32,9 +32,36 @@
   <description>Beam SDK Java IO provides different connectivity components
   (sources and sinks) to consume and produce data from systems.</description>
 
+  <properties>
+    <hadoop.version>2.7.3</hadoop.version>
```

Comment:
Also, why upgrade from 2.7.0? Do you need a new feature?

(Otherwise, it's probably better to stay at earlier version to assure maximum compatibility with provided SDKs).